services/scx_central.service

```diff
@@ -6,7 +6,9 @@ ConditionPathExists=/sys/kernel/debug/sched/ext
 Type=simple
 ExecStart=scx_central
 Restart=always
-StandardOutput=null
+StandardOutput=journal
+StandardError=journal
```

Comment:
Aren't these the default?

Reply:
To be honest, I wasn't sure - I preferred to give just in case